python/ndstorage/ndtiff_file.py

```diff
@@ -300,6 +335,8 @@ class SingleNDTiffReader:
     FOURTEEN_BIT_MONOCHROME = 5
     ELEVEN_BIT_MONOCHROME = 6
 
+    ZLIB_COMPRESSED = 8
+    NO_COMPRESSION = 1
```

Comment:
This seems redundant to `UNCOMPRESSED`. If `NO_COMPRESSION` is a standard name, maybe good to remove uncompressed